packages/api/src/derives/index.ts

```diff
@@ -24,7 +24,7 @@ import * as session from './session';
 import * as staking from './staking';
 import * as balances from './balances';
 
-export type DeriveFunc = (instanceId: string, api: ApiInterfaceRx) => (...args: CodecArg[]) => Observable<any>;
+export type DeriveFunc = (instanceId: string, api: ApiInterfaceRx) => (...args: any[]) => Observable<any>;
```

Comment:
The arguments can't be of type CodeArg always...
https://github.com/KarishmaBothara/api/blob/772769c981dbf7d7e051c59fbd91b1bc9bb67312/packages/api-derive/src/index.ts#L42